lib/graphql_metrics/analyzer.rb

```diff
@@ -0,0 +1,193 @@
+module GraphQLMetrics
+  class Analyzer < GraphQL::Analysis::AST::Analyzer
+    FieldTimingCallback = Struct.new(:static_metrics, :callback, keyword_init: true)
+
+    CONTEXT_NAMESPACE = :graphql_metrics_analysis
+    RUNTIME_METRICS_ENABLED = :runtime_metrics_enabled
```

Comment:
It's a key used in the `graphql_metrics_analysis` context namespace in order to make code that otherwise expects runtime values still work (by checking `runtime_metrics_enabled?`) when the analyzer is used without instrumentation/tracing (in other words, when someone just wants static metrics).